h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -791,6 +792,18 @@ public RegressionModelPrediction predictRegression(RowData data, double offset)
     return p;
   }
 
+  public KLimeModelPrediction predictKLime(RowData data) throws PredictException {
+    double[] preds = preamble(ModelCategory.Regression, data);
```

Comment:
Shouldn't I change this to `preamble(ModelCategory.KLime, data);` @michalkurka ?

Reply:
I think that makes sense

Reply:
great, just changed. thx!